de_workloads/data_processing/silver/spark_jobs/silver.py

```diff
@@ -1,10 +1,42 @@
-from pysparkle.pysparkle_cli import cli
+import logging
 
+from pysparkle.etl.etl import get_spark_session_for_adls, save_files_as_delta_tables
+from pysparkle.logger import setup_logger
+from pysparkle.storage_utils import get_adls_directory_contents
+from pysparkle.utils import filter_files_by_extension
 
-def call_pysparkle_entrypoint():
-    """Execute silver processing in Pysparkle."""
-    cli(["silver", "--dataset-name", "movies_dataset"], standalone_mode=False)
+
+BRONZE_CONTAINER = "raw"
+SILVER_CONTAINER = "staging"
```

Comment:
Thinking if these should be `SOURCE_CONTAINER` and `DESTINATION_CONTAINER` to make more generic (as this pattern could be used for any transformation / processing spark e.g. silver -> silver, silver -> gold)

Reply:
I updated the function parameters to `source_container` and `target_container` and removed the defaults, but I think having bronze and silver explicitly given here will be better for the templating, as in the template for silver processing we will need something like:
```
BRONZE_CONTAINER = {{ bronze_container }}
SILVER_CONTAINER = {{ silver_container }}
```
If we had source and destination here, the user generating the templates would need to think too much about what is source and what is destination.

Reply:
I'm with you on this, I like the flexibility of a single config file. Resolved.